### PR TITLE
Upgrades for arm64

### PR DESCRIPTION
**Pull request by deepsm007**
Adding additional arm upgrade jobs for 4.y.x -> 4.y.x+1

/cc @r4f4 @yselkowitz

### CODE REVIEW

File: ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.11.yaml [hunk 1]
```diff
@@ -158,6 +158,18 @@ tests:
       COMPUTE_NODE_TYPE: m6g.xlarge
       TEST_ARGS: --from-repository quay.io/multi-arch/community-e2e-images
     workflow: openshift-e2e-aws
+- as: ocp-e2e-upgrade-aws-arm64
+  interval: 72h
+  steps:
+    cluster_profile: aws-arm64
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
```
Comment (r4f4) [inline ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.11.yaml:166]: Shouldn't this be
```suggestion
      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-initial
      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
```
Or will `release:initial` and `release:latest` be properly resolved to their arm64 counterparts?

**Review by r4f4 — CHANGES_REQUESTED** [review]

**Comment by r4f4** [comment]
Nit: I see other upgrade jobs are named `e2e-$platform-upgrade`. So we could at least drop the `ocp-` prefix and maybe call it `e2e-aws-arm64-upgrade`.

And I guess the 4.12 job is failing because `release:arm64-{initial,release}` don't exist yet?

**Comment by deepsm007** [comment]
> Nit: I see other upgrade jobs are named `e2e-$platform-upgrade`. So we could at least drop the `ocp-` prefix and maybe call it `e2e-aws-arm64-upgrade`.
> 
> And I guess the 4.12 job is failing because `release:arm64-{initial,release}` don't exist yet?

ocp prefix are used to determine any jobs that are periodic and not part of `ci-operator/config/openshift/release` for collecting data in prow & testgrid

**Pull request merged**